### DISCUSSION TITLE
macOS version?

### DISCUSSION THREAD

**Discussion by eisams** [organicmaps/organicmaps#7609] [Q&A] [answered]
Is there a macOS version?
I see some references to that, but the version I have downloaded from the App Store is designed for iPad

If not, is it worth filing bugs on the iPad version run on macOS? For some reason, pinch to zoom doesn't work on the trackpad

**Comment by biodranik** [comment, marked as answer]
https://github.com/organicmaps/organicmaps/issues/7130 ?

Desktop version for Mac is used for development, it's better to use the iPad one.
